package.json

```diff
@@ -89,6 +89,7 @@
     "react-icons": "5.0.1",
     "react-modal-sheet": "2.2.0",
     "react-popper": "2.3.0",
+    "react-custom-scrollbars-2": "4.5.0",
     "react-select": "5.8.0",
     "react-use-measure": "2.1.1",
```

Comment:
"react-use-measure" can be removed since it is not used anymore

Reply:
done, nice catch!